### PR TITLE
[TT-7491] Implement transform response body OAS conversion

### PR DESCRIPTION
**Pull request by furkansenharputlu**

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5309/merge
Commit:  
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5521462013)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5309/merge
Commit: d01a9deac3e75db9a586728ecd972b0002da03e5 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5521592109)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5309/merge
Commit: e1cb4098d460d0a247d88ae8705e104ce9588537 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5541773732)

File: apidef/oas/operation.go [hunk 1]
```diff
@@ -116,6 +120,7 @@ func (s *OAS) fillPathsAndOperations(ep apidef.ExtendedPathsSet) {
 	s.fillAllowance(ep.Ignored, ignoreAuthentication)
 	s.fillTransformRequestMethod(ep.MethodTransforms)
 	s.fillTransformRequestBody(ep.Transform)
```
Comment (titpetric) [inline apidef/oas/operation.go:122]:
```suggestion
	s.fillTransformRequestBody(ep.TransformRequest)
```

This is what I was expecting here, naming?

File: apidef/oas/middleware.go [hunk 2]
```diff
@@ -781,20 +781,20 @@ func (tm *TransformRequestMethod) ExtractTo(meta *apidef.MethodTransformMeta) {
 	meta.ToMethod = tm.ToMethod
 }
 
-// TransformRequestBody holds configuration about body request transformations.
-type TransformRequestBody struct {
-	// Enabled enables transform request body middleware.
+// TransformBody holds configuration about request/response body transformations.
```
Comment (titpetric) [inline apidef/oas/middleware.go:784]:
```suggestion
// TransformBody holds configuration for request and response body transformations.
```

**Review by titpetric — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5309/merge
Commit: f1a2149f9ed1c01b7c598fe7ea1055354b494625 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5542170991)

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5309/merge
Commit: ed10c1ab4b84be645838ea859258d9e3ceb94c42 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5543062622)

**Pull request merged**